### PR TITLE
Comment out any Linux and Wayland checks

### PR DESCRIPTION
**Pull request by orowith2os**
Using appendSwitch properly causes ArmCord to segfault, and the methods shown here is nonfunctioning. I'm keeping the code there for packager reference.

### CODE REVIEW

**Comment by ghost** [comment]
pretty sure this is the original way of appending switches? - which does not append the switches (at least from my testing from before) and would make this kind of pointless
this PR should be draft until it's 100% working and tested

also when/if armcord is rewritten and using gluon this just won't be needed so the 3.X.X might aswell just run in xwayland without screenshare as it turns deprecated then imo

**Comment by smartfrigde** [comment]
ArmCord 4.0 (Gluon) is months away, it would be cool to make a temporary solutution, even with distros packing wrappers for ArmCord binary to inject those Wayland flags

File: src/main.ts [hunk 1]
```diff
@@ -25,21 +25,22 @@ if (!app.requestSingleInstanceLock()) {
...
-                console.log("Wayland specific patches applied.");
-                app.commandLine.appendSwitch("ozone-platform=wayland");
-                if (process.env.XDG_CURRENT_DESKTOP == "GNOME") {
-                    app.commandLine.appendSwitch("enable-features=UseOzonePlatform,WaylandWindowDecorations");
-                } else {
-                    app.commandLine.appendSwitch("enable-features=UseOzonePlatform");
-                }
-            }
+    // We use toLowerCase to account for desktops where XDG_SESSION_TYPE might be Wayland and not wayland.
+    if (process.platform.toLowerCase() === "linux" && process.env.XDG_SESSION_TYPE?.toLowerCase() === "wayland") {
```
Comment (ghost) [inline src/main.ts]:
```suggestion
    if (process.platform === "linux" && process.env.XDG_SESSION_TYPE?.toLowerCase() === "wayland") {
```
this'll be always lowercase "linux" on linux regardless so

**Comment by orowith2os** [comment]
I'll comment out these checks and we can leave the work to the packages to add a wrapper script, until the segfaults are fixed with the proper way of appending flags.

**Pull request merged**